9.1/Dockerfile

```diff
@@ -49,7 +49,6 @@ RUN mkdir -p /var/run/postgresql && chown -R postgres /var/run/postgresql
 
 ENV PATH /usr/lib/postgresql/$PG_MAJOR/bin:$PATH
 ENV PGDATA /var/lib/postgresql/data
-VOLUME /var/lib/postgresql/data
```

Comment:
I would replace `VOLUME` with `RUN mkdir -p`.  The `VOLUME` command also has the side-effect of creating the mount point directory if it doesn't already exist.